lib/fileio/translators/translatorMayaReference.cpp

```diff
@@ -130,6 +130,22 @@ namespace {
     name = MString(name.asChar() + 1);
```

Comment:
Can't leave the comment at line 128...but what was suggested:
```
std::replace(name.asChar(), name.asChar() + name.length(), '|', '_');
```

Mutating data like thing under an object is not recommended. You never know how data is managed and what other data may live alongside.

Reply:
Single Responsibility Principle. If your string class is doing more than simply being a string, it's probably not the best design. It might be worth in that case adding an overload for MString::replace that takes char instead of char* ?